### DISCUSSION TITLE
Where are my config files stored?

### DISCUSSION THREAD

**Discussion by EdenBendheim** [cymian/mouseless#102] [Q&A]
I just want to know where they are. Thanks :)

**Comment by cymian** [comment]
hi @EdenBendheim, I apologize for the slow reply on this -- configs are stored at `~/Library/Containers/net.sonuscape.mouseless/Data/.mouseless/configs`.  You can open it quickly (in finder) via the `Open config folder` entry from the mouseless menu.

Please let me know if you have any more questions!
